content/benefits/benefits-documents/release-notes/2024-05-22.md

```diff
@@ -0,0 +1,11 @@
+This release includes 2 new changes to the Benefits Documents API: 
+
+In order to enable dependents of deceased Veterans to upload and search for benefits related documents, we have added a new participantId parameter to the request objects for the following endpoints:
+* /documents
+* /documents/search 
+
+The participantId value is used to identify veterans and/or their dependents, and will be used instead of a file number. 
+ 
+Additionally, the Benefits Documents API now supports the upload of POA documents (VA Forms 21-22 and 21-22a). POA uploads will differ in that the claim ID field will not be a required parameter when submitting.   
```

Comment:
For the part about claim ID, what will it differ from?  Is it from what's required for other types of document uploads?

Reply:
Yes, document uploads typically require a claim ID in order to associate it to the proper claim. In the case of POA documents, those are submitted prior to a claim being established, so there's no claim ID to associate to yet.

Reply:
Ok, got it.  I see in the schema it's a required field.  I'm not sure how a user would understand when they do and don't need to provide that w/o some kind of additional details (in addition to the release note)

Reply:
We can look into how to best communicate that. We don't want users to think they don't need a claim ID, because they pretty much always will. POA is a unique case, where our only intended consumer at this time is the Benefits Claims API.